core/src/main/java/io/questdb/griffin/engine/functions/groupby/FirstVarcharGroupByFunction.java

```diff
@@ -82,6 +82,11 @@ public String getName() {
         return "first";
     }
 
+    @Override
+    public int getSampleByFlags() {
+        return GroupByFunction.SAMPLE_BY_FILL_ALL;
```

Comment:
`fill(linear)` does not work with varchar functions. (what would be even the meaning?)

good thing is that it returns a sensible error since there is a redundant mechanism to validate this, but this override is misleading.

if it's OK with you, I will remove it. the same goes for all `StrFunction` which overrides it.

Reply:
i think the flags might need to be more granular here. I added this because of failing tests. Feel free to experiment